autoPyTorch/api/base_task.py

```diff
@@ -145,6 +145,13 @@ class BaseTask(ABC):
             name and Value is an Iterable of the names of the components
             to exclude. All except these components will be present in
             the search space.
+        resampling_strategy resampling_strategy (Union[CrossValTypes, HoldoutValTypes, NoResamplingStrategyTypes]),
```

Comment:
Probably, we can define `Union[CrossValTypes, HoldoutValTypes, NoResamplingStrategyTypes]` in resampling_strategy.py so that we can easily modify (e.g. the type name, add another resampling type) in the future?

Reply:
yes I agree.